src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -1000,9 +1013,11 @@ g.test('validation_scope,no_draw_or_dispatch')
     pass.setBindGroup(1, bindGroup1);
     pass.endPass();
 
+    // Resource usage validation scope is defined by dispatch calls. If dispatch is not called,
+    // we don't need to do resource usage validation and no validation error to be reported.
     t.expectValidationError(() => {
       encoder.finish();
-    });
+    }, compute);
```

Comment:
Isn't this inverted?
```suggestion
    }, !compute);
```

Reply:
True. Success equals to compute.